app/middleware/helpers.ts

```diff
@@ -1,25 +1,26 @@
-// @flow
-
-import fs from 'fs';
-import path from 'path';
-import { app } from 'electron';
-import type { Action, Dispatch, Store } from '../reducers/types';
-import { PRINT_TO_PDF, SEND_MAIL } from '../constants/actionTypes';
-import sendMail from '../utils/email';
-import { sleep, timeout } from '../utils/async';
-import { electronRouting } from '../actions/electron';
-import ElectronUtils from '../utils/electronUtils';
-import { electronObjects } from '../store/electronObjects';
-
+import fs from "fs";
+import path from "path";
+import { app } from "electron";
+import type { Action, Dispatch, Store } from "../reducers/types";
+import { PRINT_TO_PDF, SEND_MAIL } from "../constants/actionTypes";
+import sendMail from "../utils/email";
+import { sleep, timeout } from "../utils/async";
+import { electronRouting } from "../actions/electron";
+import { electronObjects } from "../store/electronObjects";
 const pdfFolderPath = path.join(app.getPath('userData'), 'pdf');
+
 if (!fs.existsSync(pdfFolderPath)) {
   fs.mkdirSync(pdfFolderPath);
-}
+} // eslint-disable-next-line no-unused-vars
```

Comment:
eslint rule moved to wrong line

Reply:
this type of error should be caught by eslint if we (re)enabled "no unnecessary disable ..."